engine/swarm/manage-nodes.md

```diff
@@ -6,10 +6,10 @@ title: Manage nodes in a swarm
 
 As part of the swarm management lifecycle, you may need to view or update a node as follows:
 
-* [list nodes in the swarm](manage-nodes.md#list-nodes)
```

Comment:
did we fix this in Jekyll? Otherwise the `manage-nodes.md` is still needed :cry: